# Topics names of planning node
topic_map: '/vision_cone_detector/estimated_map'
topic_route: 'route'


# Hiperparameters of delaunay detector
W_DISTANCE: 1
W_ANGLE: 5
MAX_COST1: 10
MAX_COST2 : 1
MAX_DISTANCE: 8

# Parameters of spline aproximation of path
NUM_POINTS: 33
DEGREE: 3
